ui/views.py

```diff
@@ -151,7 +151,7 @@ def add_sample():
         r = requests.post(api_url + '/neural/sample/add',
                           headers=headers,
                           auth=(session['user']['token'], 'unused'),
-                          files=files)
+                          data=form.archived_images.data)
```

Comment:
```files``` variable is unused now, i think, you may remove it
And in ```recognize``` function you need to do the same change